src/mainboard/starfive/visionfive2/bt0/Cargo.toml

```diff
@@ -16,3 +16,12 @@ spin = "0.9"
 
 layoutflash = { path = "../../../../lib/layoutflash" }
 log = { path = "../../../../lib/log" }
+
+[dependencies.jh71xx-pac]
+git = "https://github.com/rmsyn/jh71xx-pac"
+version = "0.2"
+
+[features]
+default = ["12a"]
+12a = ["jh71xx-pac/visionfive2-12a"]
+13b = ["jh71xx-pac/visionfive2-13b"]
```

Comment:
Do the two boards have different SoCs?
The PAC should usually be agnostic of the board, only SoC specific, i.e., excluding extra components attached to the SoCs external buses. That would be for a potential BSP (board support package).
See also: https://docs.rust-embedded.org/discovery/microbit/04-meet-your-hardware/terminology.html

Reply:
> Do the two boards have different SoCs?

No, however the VisionFive2 13b has different ethernet peripherals. I also left it open to potentially add a `JH7100` implementation. I may drop the feature for `13b`.

Reply:
Yea, just go with one single PAC. The PHY is not considered a peripheral for the PAC, since it is external - so it goes in a potential BSP - see the aforementioned article from Rust Embedded. :-)